contracts/voting_escrow/README.md

```diff
@@ -50,58 +54,26 @@ Withdraws whole amount of veANC if lock is expired.
   "withdraw": {}
```

Comment:
we added `amount` as a parameter of `withdraw`

Reply:
@cdocu11 addressed!